dev-server/documents/html/vitalsource-pdf-page.mustache

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html><html><head>
+  <title>Principles of Economics: Chapter&nbsp;2.&nbsp;Choice in a World of Scarcity</title>
+<style>
+/*
+   Customizations from original source of page for Hypothesis dev server.
+   The only other change is to set the URL of the PDF page image.
+ */
+.screen-reader {
+  display: none;
+}
+
+#pbk-page {
+  width: 100%;
+}
+</style>
+  <script type="text/javascript">
```

Comment:
I think it would be better to format it with prettier.

Reply:
> Presuming everything here and below is untouched?

Almost, except for the URL of the `<img>` tag.

> I think it would be better to format it with prettier.

I was originally thinking to leave this "as-is" in case anyone ever wants to try diffing it against the latest version of the HTML for this page extracted from VitalSource. Currently the Prettier-formatted version is 24,000 lines long and ~730KB of text (fortunately text that compresses very well) due to the extremely long `glyphs` array that is rendered into the HTML.